.github/workflows/main.yml

```
@@ -0,0 +1,65 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python:
+        # Build on pre-releases until stable, then stable releases.
+        # actions/setup-python#213
+        - ~3.7.0-0
+        - ~3.10.0-0
+        - ~3.11.0-0
+        platform:
+        - ubuntu-latest
+        - macos-latest
+        - windows-latest
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install tox
+        run: |
+          python -m pip install tox
+      - name: Run tests
+        run: tox
```

The indentation here is inconsistent with other parts and differs from what the yamllint is set up to require:
```suggestion
    - uses: actions/checkout@v3
    - name: Setup Python
      uses: actions/setup-python@v3
      with:
        python-version: ${{ matrix.python }}
    - name: Install tox
      run: |
        python -m pip install tox
    - name: Run tests
      run: tox
```

---

(turned out that the config I committed long ago was broken and https://github.com/cherrypy/cherrypy/pull/1969 fixes that)